### PR TITLE
Card

### LINKED ISSUE #16: Update: card.html
adding sidebar contents in card.html

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. vertical Card
verticalCard
d
2. Horizontal card
Horizontal Card
3. Card with badge
BadgeCard
4. card with text overlay
TextOverlayCard
5. text only card
TextOnlyCard
6. Card with dismiss
CardDismiss
7. Card with shadow
ShadowCard

### CODE REVIEW

File: css/components/card.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
+/* === css for cards === */
+
+.card{
+    display: flex;
+    flex-direction: column;
+    border: .1rem solid rgb(128, 122, 122);
```
Comment (rahulcs754) [inline css/components/card.css:10]: you should use colour variable

File: css/components/card.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+}
+.card-bottom{
+    display: flex;
+    flex-direction: column;
+    flex: 0.5;
+    padding: .3rem;
+}
+.card-btn{
+    display: block;
+    /* display: flex; */
```
Comment (rahulcs754) [inline css/components/card.css]: remove this comment line

**Review by rahulcs754 — COMMENTED** [review]
i checked you code all is good and just few changes

File: css/components/card.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    background-color: var(--accent);
+    color: #fff;
```
Comment (tanishq20) [inline css/components/card.css]: use var()

File: css/components/card.css [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+.card-horizontal-btn{
+    margin-top: auto;
+}
+
+/* == css for badges with cards == */
+.card-badge{
+    position: absolute;
+    padding: .2rem;
+    background-color: var(--danger);
+    color: #fff;
```
Comment (tanishq20) [inline css/components/card.css]: same use var()
(Apply in all)

**Review by tanishq20 — COMMENTED** [review]
Suggested small change. Else everything looks good. 
I feel like card width is too much.

File: css/components/card.css (hunk 4 above)
Comment (RahulSingh9131) [reply css/components/card.css]: sure will try to use var()

File: css/components/card.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/components/card.css:10]: actually I have already made my color Palette, so just to give border a color , I have used the RGBA declaration.

File: css/components/card.css (hunk 2 above)
Comment (RahulSingh9131) [reply css/components/card.css]: yup !! removed that.

**Pull request merged**